src/js/jsx/shared/Droppable.jsx

```diff
@@ -45,7 +44,7 @@ define(function (require, exports, module) {
      * @param {function} getProps function to return an object with the props required by Droppable
      * @return {ReactComponent}
      */
-    var createWithComponent = function (Component, getProps) {
+    var createWithComponent = function (Component, getProps, isEqual, shouldUpdate) {
         var Droppable = React.createClass({
```

Comment:
It might make more sense for `Droppable` to be an "abstract class" that is "subclassed" by concrete implementations, and `React.createClass` would be applied to those prototype-derived objects? Seems plausible that I'm missing something though which would prevent that...